src/test/librbd/exclusive_lock/test_mock_PreReleaseRequest.cc

```diff
@@ -65,6 +92,7 @@ class TestMockExclusiveLockPreReleaseRequest : public TestMockFixture {
 public:
   typedef ImageDispatch<MockTestImageCtx> MockImageDispatch;
   typedef PreReleaseRequest<MockTestImageCtx> MockPreReleaseRequest;
+  typedef librbd::cache::rwl::ShutdownRequest<MockTestImageCtx> MockShutdownRequest;
```

Comment:
Hmm -- does this test never exercise the shutdown of the RWL cache?

Reply:
Right, the image cache's shutdown was never checked. Will add it.